.github/workflows/release-generic-actors.yaml

```
@@ -52,25 +61,63 @@ jobs:
                 settings:
                     - actor: web-scraper
+                      stable-version: 3.0
+                      stable-build-tag: version-3
+                      development-version: 0.0
+                      development-build-tag: development
                       should-build: ${{ github.event.inputs.web-scraper }}
                     - actor: cheerio-scraper
+                      stable-version: 3.0
+                      stable-build-tag: version-3
+                      development-version: 0.0
+                      development-build-tag: development
                       should-build: ${{ github.event.inputs.cheerio-scraper }}
                     - actor: playwright-scraper
+                      stable-version: 1.0
+                      stable-build-tag: version-1
+                      development-version: 0.0
+                      development-build-tag: development
                       should-build: ${{ github.event.inputs.playwright-scraper }}
                     - actor: puppeteer-scraper
+                      stable-version: 3.0
+                      stable-build-tag: version-3
+                      development-version: 0.0
+                      development-build-tag: development
                       should-build: ${{ github.event.inputs.puppeteer-scraper }}
                     - actor: jsdom-scraper
+                      stable-version: 0.1
+                      stable-build-tag: latest
+                      development-version: 0.0
+                      development-build-tag: development
                       should-build: ${{ github.event.inputs.jsdom-scraper }}
                     - actor: camoufox-scraper
+                      stable-version: 3.0
+                      stable-build-tag: version-3
+                      development-version: 0.0
+                      development-build-tag: development
                       should-build: ${{ github.event.inputs.camoufox-scraper }}
         steps:
             - uses: actions/checkout@v4
 
-            - name: Build ${{ matrix.settings.actor }}
-              uses: apify/push-actor-action@master
+            - name: Calculate versions and tags
+              id: calculate-versions
               if: matrix.settings.should-build == 'true'
-              with:
-                  token: ${{ secrets.APIFY_ACTOR_BUILD_TOKEN }}
-                  build-tag: ${{ github.event.inputs.build-tag }}
-                  version: ${{ github.event.inputs.version }}
-                  working-directory: packages/actor-scraper/${{ matrix.settings.actor }}
+              run: |
+                  if [ "${{ github.event.inputs.build-channel }}" == "stable" ]; then
+                      echo "version=${{ matrix.settings.stable-version }}\n" >> $GITHUB_ENV
+                      echo "build-tag=${{ matrix.settings.stable-build-tag }}\n" >> $GITHUB_ENV
+                  elif [ "${{ github.event.inputs.build-channel }}" == "development" ]; then
```

```suggestion
                  if [ "${{ github.event.inputs.build-channel }}" = "stable" ]; then
                      echo "version=${{ matrix.settings.stable-version }}\n" >> $GITHUB_ENV
                      echo "build-tag=${{ matrix.settings.stable-build-tag }}\n" >> $GITHUB_ENV
                  elif [ "${{ github.event.inputs.build-channel }}" = "development" ]; then
```

You're supposed to use `=` for string comparison, `==` is a non-standard extension :imp: